lib/Fmt_ast.ml

```diff
@@ -4342,9 +4350,17 @@ and fmt_let c ~ext ~rec_flag ~bindings ~parens ~fmt_atrs ~fmt_expr ~body_loc
        $ hvbox 0 fmt_expr ) )
   $ fmt_atrs
 
-and fmt_value_binding c ~rec_flag ?ext ?in_ ?epi
+and fmt_value_binding c ~rec_flag ?ext ?in_ ?loc_in ?epi
     {lb_op; lb_pat; lb_args; lb_typ; lb_exp; lb_attrs; lb_loc; lb_pun} =
-  update_config_maybe_disabled c lb_loc lb_attrs
+  let loc_with_in =
```

Comment:
It would be nice to have a comment saying what `loc_with_in` means and in which cases `loc_in` is expected to be `None`.